.github/PULL_REQUEST_TEMPLATE/feature.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature (PR)
```

Comment:
I think we can have a single template for PRs for the time being, since the content you suggest is almost the same. Remove one of the templates?

Reply:
agreed.